hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakePeriodManager.java

```diff
@@ -107,17 +108,17 @@ public boolean isRewardable(
     /**
      * Returns the first stake period that is not rewardable. This is used to determine
      * if an account is eligible for a reward, as soon as staking rewards are activated.
-     * @param networkRewards the network rewards
+     * @param rewardsStore the network rewards store
      * @param consensusNow the current consensus time
      * @return the first stake period that is not rewardable
      */
     public long firstNonRewardableStakePeriod(
-            @NonNull final ReadableNetworkStakingRewardsStore networkRewards, @NonNull final Instant consensusNow) {
+            @NonNull final ReadableNetworkStakingRewardsStore rewardsStore, @NonNull final Instant consensusNow) {
 
         // The earliest period by which an account can have started staking, _without_ becoming
```

Comment:
I had to read this comment three times 😝...if we do want a comment here, I think a small conceptual refresher like,
```
// Remember that accounts are only rewarded for _full_ periods; so if Alice started staking in the
// previous period (current - 1), she will not have completed a full period until current has ended
```
...would be more helpful.

Reply:
Added this comment also 😛 . Thank you !